### ISSUE TITLE
Finalize two creative tests

### ISSUE THREAD

**Issue by brettgaylor** [MozillaFoundation/Advocacy#679]
Detailed description of work: We have two pieces of creative we started in our message testing session.  Let's finish these and get them in shape for facebook tests.

Link to creative 1:
https://drive.google.com/drive/folders/0BzuBOR2VJQqUUGR5dHp4cUNhczQ

Link to creative 2:
https://drive.google.com/file/d/0B0bF99sa1fRJT2lPTEN4WXdvb0U/view?usp=sharing

*Start date: December 15th, 2016* 
*Due date: December 16th, 2016* 
Note: The dates need to be exactly in the format indicated above for proper integration with the Gantt chart.

CC:

**Comment by beccaklam** [comment]
Hey @brettgaylor can you give a bit more vision/clarity about what you're hoping for these two pieces of creative? And are we waiting for @lovegushwa to come back before finishing?

**Comment by brettgaylor** [comment]
@HPaulJohnson can you clarify - do we need two creatives for each concept, or just two creatives with different text?  I'm looking at the [spreadsheet of splits](https://docs.google.com/spreadsheets/d/1ip-MDUA_RMtIJGorYiE1ayG-TNUgSF_VmY3G6IOlzE4/edit#gid=0) and not clear to me.  We have two pieces of creative that we can use this ticket to advance, but lmk if you need more.

**Comment by beccaklam** [comment]
@brettgaylor @HPaulJohnson The creative (rough, WIP) that we have so far: https://redpen.io/p/bc23d9f2d0dec4f109

**Comment by brettgaylor** [comment]
@beccaklam as discussed, here's the 2nd text for the embarrassing texts creative 
https://docs.google.com/document/d/1Fu85oel_pgvHxCfbR1ZjVhZ19ONL0taz7u48lEwKktU/edit

**Comment by beccaklam** [comment]
@brettgaylor @lovegushwa @HPaulJohnson New creative added as we discussed yesterday. I did the in-situ with the hand but then the text is quite small? https://redpen.io/p/bc23d9f2d0dec4f109

**Comment by brettgaylor** [comment]
Sweet!  Yeah it does look hard to read doesn't it?  Should we just go with
the two autocorrect fails minus the hand?

Thanks becca! :100:

**Comment by beccaklam** [comment]
@brettgaylor I think so. Having it be just the screen shot allows it to be more easily read and focuses the viewer on the text.

**Comment by brettgaylor** [comment]
🕶

**Comment by HPaulJohnson** [comment]
Works for me. 

@beccaklam can you send me the files for the 2 eye versions and the 2 text fail versions and i can get these fired up on Facebook.

**Comment by beccaklam** [comment]
@HPaulJohnson Here are the files! https://drive.google.com/open?id=0BzuBOR2VJQqUUGR5dHp4cUNhczQ

**Comment by lovegushwa** [comment]
@beccaklam these looks great. Excited about this test :)

**Comment by HPaulJohnson** [comment]
These tests have officially been launched on Facebook. Final matrix here: 

https://docs.google.com/spreadsheets/d/1ip-MDUA_RMtIJGorYiE1ayG-TNUgSF_VmY3G6IOlzE4/edit#gid=0

We ended up doing two designs for each campaignable concept, using the same copy across designs. I also added in Pizza Cat, which performed well in earlier social fundraising test, as a control against these new creatives. 

I'll report back soon as we start to see some results.
